met-api/tests/unit/api/test_engagement.py

```diff
@@ -64,6 +64,23 @@ def test_get_engagements(client, jwt, session, engagement_info):  # pylint:disab
     assert created_eng.get('content') == rv.json.get('content')
 
 
+@pytest.mark.parametrize('engagement_info', [TestEngagementInfo.engagement1])
+def test_get_engagements_paginated_advanced_search(client, jwt,
+                                                   session, engagement_info):  # pylint:disable=unused-argument
+    """Assert that an engagement can be fetched."""
```

Comment:
"""Assert that an engagement can be fetched."""

Please add the real purpose of the test.

"""Assert that an engagement can be fetched by filtering using the engagement status."""

Reply:
updated, thanks